public/app/features/scenes/core/SceneObjectBase.tsx

```diff
@@ -26,6 +33,8 @@ export abstract class SceneObjectBase<TState extends SceneObjectState = {}> impl
   isActive?: boolean;
   events = new EventBusSrv();
 
+  // this is not working
+  // constructor(state: Defaultize<TState, ExtractDefaultPropsType<this>>) {
```

Comment:
not been able to figure this out, want to be able to defer the defaults type from a static defaultProps property defined on the class type